main.py

```diff
@@ -305,16 +306,93 @@ def write_changelog(repo, changelog, path, commit_message):
     repo.update_file(contents.path, commit_message, changelog, contents.sha)
 
 
+def get_inputs_from_file(file_name, input_name):
+    '''
+    Read configuration from given local workflow file
+
+    Args:
+        file_name (string): the file to read from
+        input_name (string): input_name in workflow file
+
+    Returns:
+        string: action_input
+    '''
+    file = open(file_name, 'r', encoding='utf-8')
+    file_content = file.read().split('\n')
+    file.close()
+    target_flag = 0
+    for _, line in enumerate(file_content):
+        if target_flag == 0 and re.search('auto-generate-changelog@master',
+                                          line):
+            target_flag = 1
+        if target_flag == 1 and re.search(input_name, line):
+            return re.search("'(.*)'", line).group().strip('\'')
```

Comment:
You may search an input of another section not belong to `auto-generate-changelog` section by this way.